packaging/docker/update_reqs.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-pip-compile ../../setup.py -Uo requirements.txt
```

Comment:
This is needed to not pollute the generated requirements.txt file with indexes specified in the user's pip configuration